frontend/src/pages/devices/List.tsx

```diff
@@ -143,15 +147,12 @@ const List = () => {
     warnings.push.apply(warnings, getLastWentOfflineWarnings(entity))
     warnings.push.apply(warnings, getLockResponsivenessWarnings(entity))
 
-    if (warnings.length === 1) {
-      return <>{warnings[0]}</>
-    }
     return (
-      <ul>
+      <ListGroup className="flush">
         {warnings.map((warn) => (
-          <li>{warn}</li>
+          <ListGroup.Item className="border-0">{warn}</ListGroup.Item>
```

Comment:
What does this look like w/ and w/o the `border-0`?

Reply:
It gives the warnings this weird border. Do we want this?
![image](https://user-images.githubusercontent.com/54916525/199120389-4387bfb6-416f-4d1b-b3ad-fc5393062af8.png)

Reply:
What if we got rid of the `label` and used a [contextual class](https://getbootstrap.com/docs/4.0/components/list-group/#contextual-classes) instead?

Reply:
> > What does this look like w/ and w/o the border-0?
> 
> ?

The image above is what it is without the border-0, this is what it looks like with it:
![image](https://user-images.githubusercontent.com/54916525/200089007-70d77b00-f1b4-4456-8685-62c8bc6611ac.png)

Reply:
I wonder if the way we're doing the `flush` is wrong (it doesn't look like [this](https://getbootstrap.com/docs/4.0/components/list-group/#flush)).

I think it might be good to keep the border and add [contextual classes](https://getbootstrap.com/docs/4.0/components/list-group/#contextual-classes). Otherwise we'll probably fight against the system as we try to line things up across columns. Maybe try the `light` class for the non-`danger` ones.

Reply:
https://github.com/anthonywittig/mLock/pull/37/commits/202630e439117ffe5eba15c36831018e12cab4a3